### PR TITLE
Create a module to have a media query on JS

### PR DESCRIPTION
**Pull request by nathpaiva**
To know the size of the user's screen I created a module installed via npm to be able to have this information.

This module  provides some methods to know the type of screen of the client.
Was made with pre-set breakpoint.

Adds:

* Gitpage
* Semaphore integration badge
* Codeclimate integration badge
* Test Coverage integration badge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+`yarn build`
+
+
+### Usage
+
+After install, you will have to add the stylesheet (dist/media-query.css) to your page via link tag or import it to your existing style manifest.
+
+You can use it via:
+* GLOBAL variabel
```
Comment (eduardojmatos) [inline README.md]: `variable`

File: demo.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
+body {
+  font-family: 'Lato', sans-serif;
```
Comment (eduardojmatos) [inline demo.css:2]: This font will only works when Google API Fonts was used

File: package.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
+{
+  "name": "media-query-js",
+  "version": "1.0.0",
+  "description": "Media Query helper to check breakpoints via javascript",
```
Comment (eduardojmatos) [inline package.json]: `JavaScript`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  },
+  "homepage": "https://github.com/getninjas/media-query-js#readme",
+  "scripts": {
+    "test": "cross-env NODE_ENV=test && karma start",
+    "version": "npm run build && git add -A dist",
+    "postversion": "git push && git push --tags",
+    "config": "npm config set progress=false && npm config set git-tag-version=true",
+    "clear": "rm -rf ./build/*",
+    "lint": "eslint -c .eslintrc.json src/*.js",
+    "build": "npm run build:js && npm run build:js:prod",
```
Comment (eduardojmatos) [inline package.json]: I dont undestand why we are running two builds of js in one command. Maybe the names seems confused to understand what they are doing for real. My suggestion is to use some thing like:

```JSON
 "build:babel": "babel src -d dist",
 "build:js:prod": "cross-env BABEL_ENV=production babel src/media-query.js -o dist/media-query.min.js && npm run copy"
 "build:prod": "npm run build:babel && npm run build:js:prod",
```

File: package.json [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-coverage": "^1.1.1",
+    "karma-fixture": "^0.2.6",
+    "karma-html2js-preprocessor": "^1.1.0",
+    "karma-jasmine": "^1.1.0",
+    "karma-spec-reporter": "0.0.31",
+    "karma-threshold-reporter": "^0.1.15",
+    "npm-run-all": "^3.1.2",
+    "onchange": "^3.2.1",
+    "prismjs": "^1.8.1"
```
Comment (eduardojmatos) [inline package.json:69]: Do we need prismjs for what?

**Review by eduardojmatos — CHANGES_REQUESTED** [review]

File: package.json (hunk 5 above)
Comment (iondrimba) [reply package.json:69]: GitPage Docs

File: dist/media-query.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+      value: function beyondMobile() {
+        return this.isTablet() || this.beyondTablet();
+      }
+    }]);
+
+    return MediaQuery;
+  }();
+
+  exports.default = MediaQuery;
+});
```
Comment (iondrimba) [inline dist/media-query.js]: EOF

File: .gitignore [hunk 7]
```diff
@@ -1,48 +1,19 @@
-# Logs
```
Comment (williamcosta) [inline .gitignore:1]: I think it was nice to leave the extension `log` files.

File: demo.css (hunk 2 above)
Comment (nathpaiva) [reply demo.css:2]: @eduardojmatos We are using on our Demo page! Imported via link inside the `index.html`

File: .gitignore (hunk 7 above)
Comment (nathpaiva) [reply .gitignore:1]: @williamcosta We don't have any log on our module!

**Review by williamcosta — APPROVED** [review]

**Review by eduardojmatos — APPROVED** [review]

**Pull request merged**